app/editor/src/features/content/tool-bar/sections/CreateNewSection.tsx

```diff
@@ -0,0 +1,26 @@
+import { FaFileAlt, FaFileAudio, FaFileImage } from 'react-icons/fa';
+import { GiFairyWand } from 'react-icons/gi';
+import { useNavigate } from 'react-router-dom';
+
+import * as styled from './styled';
+
+/**
+ * Section containing the create new content buttons
+ * @returns Section with three separate create content buttons
+ */
+export const CreateNewSection: React.FC = () => {
+  const navigate = useNavigate();
+  return (
+    <styled.CreateNewSection
```

Comment:
I don't see a lot of value of these components being wrapped in a styled component, when the styled component is actually the component they're being wrapped in `ToolBarSection`.  It's confusing and presently there is no reason to do it.  Just wrap these components in the `ToolBarSection`.